testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/messaging/ha/AbstractMessagingHATestCase.java

```diff
@@ -151,22 +150,24 @@ protected static void checkHornetQServerStartedAndActiveAttributes(JMSOperations
     protected static InitialContext createJNDIContextFromServer1() throws NamingException {
         final Properties env = new Properties();
         env.put(Context.INITIAL_CONTEXT_FACTORY, "org.jboss.naming.remote.client.InitialContextFactory");
-        env.put(Context.PROVIDER_URL, System.getProperty(Context.PROVIDER_URL, "remote+http://127.0.0.1:8080"));
+        String ipAdddress = TestSuiteEnvironment.formatPossibleIpv6Address(System.getProperty("node0", "127.0.0.1"));
+        env.put(Context.PROVIDER_URL, System.getProperty(Context.PROVIDER_URL, "remote+http://" + ipAdddress + ":8080"));
```

Comment:
This violates RFC, [ and ] are missing.

Reply:
] and [ are added by TestSuiteEnvironment.formatPossibleIpv6Address

Reply:
Ah, ok.